### ISSUE TITLE
Date axis won`t work in PBI

### ISSUE THREAD

**Issue by CRaDeRa** [microsoft/charticulator#306]
Hi,

I`ve made a radial chart which is still rather simple, but I don`t want to progress further until I`ve made it work properly in Power BI. The issue is that everytime I put the date on the radial axis, I always get the “Dataset incomplete” message in Power BI even though all the data is provided. If I remove the date from the axis and replace it with some other attribute the chart works fine. Is there a date format issue I need to know about, or a certain code that needs to be added to the plotsegment filtering to make it work? I`ve seen examples of this working for others (ex: https://community.powerbi.com/t5/Data-Stories-Gallery/Charticulator-Radial-and-Scatter-Plots-with-Diverging-Colour/td-p/930913) so I assume I`m doing something wrong.

**Comment by doubleamp** [comment]
I can confirm that I have also had problems with date fields (see my comments to pull request #305), so for now I am not using them in my custom visuals. When I have tried in the last few days, I have had the same 'Dataset incomplete' message in PowerBI.
I have tried to upload the .chart corresponding to the link that @CRaDeRa posted, both to the charticulator site and to my own local server, and it refuses to load; it may be the format of the .chart file, or maybe some other problem related to the date fields problem.

**Comment by snapper911T** [comment]
Can you share your .chart file and data set sample?

**Comment by dm-p** [comment]
I've attempted to re-create the chart from scratch and it definitely isn't working as it should be. Previously I could add the date to the radial axis and the month component of the date to the angular axis and you'd get something like this (which I've quickly knocked-up in 1.4.0):
![image](https://user-images.githubusercontent.com/10572054/89718946-8ba26480-da17-11ea-8f19-6cfbdceff96d.png)

----
If I try to do this in 1.6.0, I get the valid expressions in the plot segment, but the chart doesn't render the glyphs correctly, e.g.:
![image](https://user-images.githubusercontent.com/10572054/89718912-3403f900-da17-11ea-95bc-0ce8db2bd005.png)

I note that in 1.4.0 - note that the expressions are the same, but the range data is a POSIX timestamp rather than formatted date/time (in 1.6.0).

If I try to type these timestamps in to see if I can override them, Charticulator will convert them in the UI back to formatted dates and they'll display for a brief second and then blank out, so I think the root cause may be around here somehwhere.

Here's the .chart file (which I note cannot be imported in 1.6.0) - have had to .zip it as GitHub doesn't support .chart as a valid extension: [global_temp_anomaly_radial.zip](https://github.com/microsoft/charticulator/files/5046162/global_temp_anomaly_radial.zip)

If I have this chart saved in my local session cookies for 1.4.0 and then attempt to load into 1.6.0, it won't actually work and the UI just sits there - there's nothing reported in the browser console that I could use to provide any further info, and other chart types I have stored seem to work just fine.
